deltachat-ios/Controller/EphemeralMessagesViewController.swift

```diff
@@ -32,7 +32,20 @@ class EphemeralMessagesViewController: UITableViewController {
         self.dcContext = dcContext
         self.chatId = chatId
         super.init(style: .grouped)
-        self.currentIndex = self.options.index(of: dcContext.getChatEphemeralTimer(chatId: chatId)) ?? 0
+
+        // select option close to the timespan (that may no be available as an option eg. in case option have changed)
```

Comment:
this is the same what we're doing on android, and it seems to be good enough, esp. as this is only for cornercases as removed options - and most chat members do not tweak this option anyways